CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.2.1
+
+_Dec 17, 2021_
+
+A big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🖨 Print export has a cleaner breaking page
+- 🧑‍💻 Add `pinnable` option
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### Changes
```

Comment:
The line with the package version is missing. BTW, in #3312 I inverted by putting the package before the highlights. In #3385 the same mistake was repeated.

```suggestion
### `@mui/x-data-grid@v5.2.1` / `@mui/x-data-grid-pro@v5.2.1`

#### Changes
```